geode-core/src/upgradeTest/java/org/apache/geode/internal/cache/rollingupgrade/RollingUpgradeDUnitTest.java

```diff
@@ -133,8 +135,9 @@ private void deleteWorkingDirFiles() {
   @Override
   public void postSetUp() throws Exception {
     deleteWorkingDirFiles();
-    IgnoredException.addIgnoredException(
-        "cluster configuration service not available|ConflictingPersistentDataException");
+    addIgnoredException("cluster configuration service not available");
+    addIgnoredException(ConflictingPersistentDataException.class);
+    addIgnoredException(DistributedSystemDisconnectedException.class);
```

Comment:
Might it be better to instead use "Abandoned because shutdown is in progress" as the ignored exception, rather than ignoring any `DistributedSystemDisconnectedException`? Or is it reasonable that any `DistributedSystemDisconnectedException` can be safely ignored in these tests?

Reply:
"Abandoned because shutdown is in progress" sounds good to me.